models/providers.go

```diff
@@ -58,4 +59,9 @@ type Provider interface {
 	PublishMetrics(tokenVal string, data *MesheryResult) error
 	GetResult(*http.Request, uuid.UUID) (*MesheryResult, error)
 	RecordPreferences(req *http.Request, userID string, data *Preference) error
+
+	SMPSTestConfigStore(req *http.Request, perfConfig *SMPS.PerformanceTestConfig) (string, error)
```

Comment:
Given the short-lived nature of the local provider, I wonder if its best to include here or not. Hmm.

Reply:
Actually the provider object would hold a bitcast storage. So we would be using that to CRUD these things. That should be fine right?